.github/workflows/codespell.yaml

```diff
@@ -20,3 +20,4 @@ jobs:
           check_filenames: true
           ignore_words_list: ExtraVersion,extraversion,ba
           check_hidden: true
+        run: CONTAINER_CMD=docker make containerized-test TARGET=codespell
```

Comment:
I do not think this is correct. You probably should just copy one of the other workflows and replace `TARGET=...` with `TARGET=codespell`. There should be no need to have the codespell options listed in this file anymore, as those are now set in the `Makefile`.

Reply:
Yes this yaml file show remain same as other yaml file in the workflows directory with only Target and name change.

Reply:
@nixpanic @Madhu-1 still there is an issue removing uses section, as the flags check_filename and check_hidden are not being mentioned in the command running in Makefile. Not sure if it's important to have that?

Reply:
I don't think that is much of a problem. For me, it is more important that the checks in CI can be run locally with the same configuration.

Reply:
I have updated the PR. Do review it.